docs/synthetics/index.md

```diff
@@ -0,0 +1,14 @@
+title: Sematext Synthetics Overview
+description: Overview of Synthetics for API, Web URL, website and user journey monitoring
+
+Sematext Synthetics provides the ability to actively monitor API, Web URLs, websites and user journeys from global locations. Synthetics help you ensure the availability and performance of your APIs and websites. While Experience helps you track real user performance of your websites, Synthetics will help you identify downtime and performance by simulating the user requests and interactions with your web page. 
+
+<< screenshots of monitors overview screen >>
+
+### How it works
+
+You start by creating a Synthetics App. Under the App, you can create an HTTP or Browser monitor. HTTP monitor can be used to monitor any HTTP endpoints like API endpoints or Web URL. HTTP monitor sends an HTTP request to the specified URL with configured request settings and records the timings and response details. Browser monitor can be used to monitor websites and user journey by writing a script. Browser monitor script will be executed in a browser and the performance is recorded.  While creating the monitor you can specify a list of conditions that will decide the result of the run.
```

Comment:
Maybe we should link "App" to https://sematext.com/docs/guide/app-guide/ ?

any HTTP endpoints like API endpoints or Web URL => any HTTP endpoints like HTTP APIs or Web URLs

by writing a script => by writing a script that simulates users actions

Browser monitor script will be executed => Browser monitor script executes